packages/lib/src/external-apis/BackendAPI.ts

```diff
@@ -33,7 +33,7 @@ export async function getChallenge(account: Account): Promise<string> {
     const checkedAccount = checkAccount(account);
     return (
         await axios.get(
-            (checkedAccount.profile.deliveryServiceUrl ?? '') +
+            (checkedAccount.profile.deliveryServices[0] ?? '') +
```

Comment:
I think we should throw an error if `checkedAccount.profile.deliveryServices[0]` is null or undefined